docs/FAQ.md

```diff
@@ -111,6 +111,71 @@ store arbitrary files in `<your-git-repo>/scratch/`.
 
 You can find more details on `gitignore` files [here][gitignore].
 
+### How can I avoid committing my local-only changes to tracked files?
```

Comment:
Nit: Shouldn't this be "How can I keep local changes private?"

And then the introduction should mention the use-cases of configs and/or build files.